addons/core/functions/fnc_getVehicleRadios.sqf

```diff
@@ -0,0 +1,36 @@
+#include "script_component.hpp"
+
+/*
+    Name: TFAR_fnc_getVehicleRadios
+
+    Author(s):
+        Jaffa
+
+    Description:
+        Gets a list of radios in the specified vehicle
+
+    Parameters:
+        0: Object - vehicle
+
+    Returns:
+         Array of LR radio arrays in format: 
+             0 - Object - Vehicle, 1 - String - Radio Settings ID
+         or empty array if no vehicle radios found
+
+    Example:
+        _radios = _vehicle call TFAR_fnc_getVehicleRadios;
+*/
+params [["_vehicle",0]];
```

Comment:
Doesn't typeCheck that you pass an object

Reply:
It does here https://github.com/michail-nikolaev/task-force-arma-3-radio/pull/1371/files#diff-07960b095b64fe84277c6f2cf1e09662R24

Reply:
Why not use the return value of params:
`If !(params [["_vehicle", objNull, [objNull]]]) exitWith {/*ERROR*/};`